### PR TITLE
CORE: Added latent ID 56

### PR DESCRIPTION
**Pull request by BRFord88**
Added latent check for weapon drawn mp over {interval} used in rune weapons.

### CODE REVIEW

File: src/map/latent_effect_container.cpp [hunk 1]
```diff
@@ -296,6 +296,16 @@ void CLatentEffectContainer::CheckLatentsMP(int32 mp)
...
+		case LATENT_WEAPON_DRAWN_MP_OVER:
+			if (mp > m_LatentEffectList.at(i)->GetConditionsValue() && m_POwner->animation == ANIMATION_ATTACK)
+			{
+				m_LatentEffectList.at(i)->Activate();
+			}
+			else
+			{
+				m_LatentEffectList.at(i)->Deactivate();
+			}
+			break;
```
Comment (xdemolish) [inline src/map/latent_effect_container.cpp]: tabs > 4 spaces in this block of code

File: src/map/latent_effect_container.cpp [hunk 2]
```diff
@@ -414,6 +424,16 @@ void CLatentEffectContainer::CheckLatentsEquip(uint8 slot)
...
+			case LATENT_WEAPON_DRAWN_MP_OVER:
+				if (m_POwner->health.mp > m_LatentEffectList.at(i)->GetConditionsValue() && m_POwner->animation == ANIMATION_ATTACK)
+				{
+					m_LatentEffectList.at(i)->Activate();
+				}
+				else
+				{
+					m_LatentEffectList.at(i)->Deactivate();
+				}
+				break;
```
Comment (xdemolish) [inline src/map/latent_effect_container.cpp]: and here

**Comment by xdemolish** [comment]
also you don't need to open a new pull request
make changes > amend last commit > force push > leave a comment when updated
github will automatically update the pull request on pushing

**Comment by BRFord88** [comment]
alright, sounds good.

**Comment by BRFord88** [comment]
Made aforementioned changes.

**Comment by xdemolish** [comment]
looks like it didn't push, try force pushing the same branch

**Comment by BRFord88** [comment]
It appears to be doing something.

File: src/map/latent_effect_container.cpp [hunk 3]
```diff
@@ -414,6 +424,16 @@ void CLatentEffectContainer::CheckLatentsEquip(uint8 slot)
...
+            case LATENT_WEAPON_DRAWN_MP_OVER:
+                if (m_POwner->health.mp > m_LatentEffectList.at(i)->GetConditionsValue() && m_POwner->animation == ANIMATION_ATTACK)
+                {
+                    m_LatentEffectList.at(i)->Activate();
+                }
+                else
+                {
+                    m_LatentEffectList.at(i)->Deactivate();
+                }
+				break;
```
Comment (xdemolish) [inline src/map/latent_effect_container.cpp]: almost!
rebase, then change this line to use spaces, amend last commit, force push

Comment (xdemolish) [inline src/map/latent_effect_container.cpp]: oh and run another rebase and squash similar commits after the first rebase (remember to force push)

**Comment by BRFord88** [comment]
Okay, I did the git push --force after amending my commit and rebasing.  I hope I got it right this time, thank you for being patient with me.

**Comment by xdemolish** [comment]
perfect!
thanks for contributing :)

**Pull request merged**